add imshow_bat

xingyu-liu opened the pull request.

Commit:
add imshow_bat

Commit:
Refine: imshow_bat

zhenzonglei commented:
Cool pull. But  I do think the funciton name is awkward. Is  imgarray_show better? or ia_show? ia = image array.

xingyu-liu commented:
> Cool pull. But I do think the funciton name is awkward. Is imgarray_show better? or ia_show? ia = image array.

good idea! 'imshow_bat' might confuse others with saving single image one by one, didn't point out the real function of displaying images matrix in one figure.

sunshineDrizzle commented:
1. 建议去掉第1行，这是个用于import的module，指定执行该文件的解释器不合理
2. docstring的格式可以整理一下
3. 加个output之类的参数，这个参数不是None的时候讲产生的figure存到output指定的文件夹下
4. 第41行可以使用enumerate

Commit:
Refine: imshow_bat

1. rename imshow_bat with imgarray_show
2. refined according to comments of Xiayu

sunshineDrizzle committed:
RF: tidy annotations
